pylatex/lists.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+"""
+    pylatex.lists
+    ~~~~~~~
+
+    This module implements the class that deals with latex List objects
+    specifically Enumerate and Itemize.
+
+    :copyright: (c) 2015 by Sean McLemon.
+    :license: MIT, see License for more details.
+"""
+
+from .base_classes import BaseLaTeXNamedContainer
+
+
+def Item(s, label=None):
```

Comment:
This is a function not a class. The Capital letter at the start suggests otherwise. I'm surprised flake8 didn't complain about it, since it's right here: https://www.python.org/dev/peps/pep-0008/#function-names